docker-compose.yml

```diff
@@ -8,10 +8,11 @@ services:
     volumes:
       - ./app/:/usr/src/app/
     ports:
-      - 127.0.0.1:8000:8000
+      - 127.0.0.1:${SERVER_PORT}:${SERVER_PORT}
     env_file: .env
     environment:
       DEBUG: 0
+      SERVER_PORT:
```

Comment:
Ah, right. Because we've set `env_file: .env` above it's passing all of the environment variables in. We aren't doing that in course-planner, so we did need to specify the individual variables like this in that project, and I think I just carried that over here.

But since it's not needed, I took it out in a0d317d.

Reply:
Yep. FYI you can also combine the approaches and use `.env` to supply most variables, then use `environment:` to create composite env vars from things in `.env` (if you weren't aware)